### ISSUE TITLE
Editor Welcome Tour: Focus on the first element when the tour is opened

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [Automattic/wp-calypso#55635]
### Why
The Welcome Tour currently does not receive focus when first opened (either via the sidebar or reloading a page with the tour opened). Ideally, the tour should receive the focus at least when loaded from the sidebar, so that keyboard navigation can be initiated immediately.

### What
Focus on the first element when the tour is first opened. Then, the user can initiate navigation (w/ keyboard) immediately.
The focus shouldn't be lost after selecting something in the tour either.

### Acceptance Criteria
- [ ] focus on the first element when tour is opened
- [ ] focus shouldn’t be lost after you select something

### Related

Improved keyboard navigation is explored in #55634
